### ISSUE TITLE
Terminations on newlines

### ISSUE THREAD

**Issue by microcosm** [evbacher/gd2md-html#17]
If I have some content in gDocs which is marked up like this (in HTML just to illustrate)

```
<strong>My bolded text<br /></strong>My regular text
```

This addon will generate this:

```
**My bolded text
**My regular text
```

Which makes sense, but it breaks the markdown interpretation based on what is really intended. It would be better if it figured that out and gave this:

```
**My bolded text**
My regular text
```

Unfortunately there is no way in gDocs for you to realize that you are committing that formatting mistake, so we can't really lump it on the gDocs editor who wrote it.

Great addon by the way, thanks for sharing!

**Comment by evbacher** [comment]
As with the other bug, we can do this. I'll close these two bugs when I get that change in.

**Comment by evbacher** [comment]
Fixed in v 1.0β10. Please reopen (or open another bug) if still a problem.
